.github/workflows/codeql-analysis.yml

```
@@ -62,7 +62,9 @@ jobs:
           languages: ${{ matrix.language }}
 
       - name: Maven Build
-        run: ./mvnw verify pmd:aggregate-pmd -Pci
+        run: ./mvnw verify pmd:aggregate-pmd -Pci  \
+          -Ddevelocity.cache.local.enabled=false \
+          -Ddevelocity.cache.remote.enabled=false
```

```suggestion
        run: ./mvnw verify pmd:aggregate-pmd -Pci \
          -Ddevelocity.cache.local.enabled=false \
          -Ddevelocity.cache.remote.enabled=false
```

I don't know why this failed on CI, it works fine locally.  Maybe it's some odd shell issue? removing the extra space here, but I don't expect this to do anything.

@clayburn any ideas?